packages/appstore/src/components/routes/routes.tsx

```diff
@@ -20,12 +24,21 @@ const Routes: React.FC = () => {
             <Switch>
                 {getRoutesConfig({
                     consumer_routes: config.routes,
-                }).map((route: TRoute, idx: number) => (
-                    <RouteWithSubroutes key={idx} {...route} />
-                ))}
+                }).map((route: TRoute, idx: number) => {
+                    // Temporary way to intercept the route to show the Wallets component.
+                    let updated_route = route;
+                    if (updated_route.path === '/appstore/traders-hub') {
+                        updated_route = {
+                            ...updated_route,
+                            component: is_next_wallet_enabled ? Wallets : updated_route.component,
```

Comment:
Switching to the `@deriv/wallets` when the `is_next_wallet_enabled` flag is enabled, If not rendering the legacy traders hub page.